pkg/server/filters/filters.go

```diff
@@ -121,7 +120,7 @@ func WithClusterScope(apiHandler http.Handler) http.HandlerFunc {
 			// fallthrough
 			cluster.Name = logicalcluster.Wildcard
 		case clusterName.Empty():
-			cluster.Name = genericcontrolplane.LocalAdminCluster
```

Comment:
if we get rid of `cluster.Wildcard`, i don't even think we need the logic in here. instead:

```go
if clusterName != logicalcluster.Wildcard {
  // do regex check
}
cluster.Name = clusterName
```

Reply:
can do in a follow-up if you want

Reply:
Didn't know we were planning to get rid of `cluster.Wildcard`. I can do that in a follow-up. 
Meanwhile, I removed the case when the `clusterName` was empty. We will make use of the default value.

Reply:
Yeah I think the same arguments used to not have a .Wildcard for shard name apply here